x-pack/legacy/plugins/infra/index.ts

```diff
@@ -70,45 +59,24 @@ export function infra(kibana: any) {
           url: `/app/${APP_ID}#/logs`,
         },
       ],
-      mappings: savedObjectMappings,
+      // mappings: savedObjectMappings,
     },
     config(Joi: typeof JoiNamespace) {
-      return getConfigSchema(Joi);
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
+      })
+        .unknown()
+        .default();
     },
     init(legacyServer: any) {
-      const { newPlatform } = legacyServer as KbnServer;
-      const { core, plugins } = newPlatform.setup;
-
-      const infraSetup = (plugins.infra as unknown) as InfraSetup; // chef's kiss
-
-      const initContext = ({
-        config: infraSetup.__legacy.config,
-      } as unknown) as PluginInitializerContext;
-      // NP_TODO: Use real types from the other plugins as they are migrated
-      const pluginDeps: InfraServerPluginDeps = {
-        home: legacyServer.newPlatform.setup.plugins.home,
-        usageCollection: plugins.usageCollection as UsageCollectionSetup,
-        indexPatterns: {
-          indexPatternsServiceFactory: legacyServer.indexPatternsServiceFactory,
+      // NP_TODO: How do we move this to new platform?
+      legacyServer.addAppLinksToSampleDataset('logs', [
```

Comment:
I think something may have potentially gone wrong with a merge somewhere along the line, and the migration from `legacyServer.addAppLinksToSampleDataset` to `plugins.home.sampleData.addAppLinksToSampleDataset` in #52753.

We can remove `legacyServer.addAppLinksToSampleDataset` (it doesn't exist in `master`).

(This file is also throwing an exception as `logsSampleDataLinkLabel` doesn't exist here).